lib/elixir/src/elixir_import.erl

```diff
@@ -157,7 +157,7 @@ is_sigil({Name, 2}) ->
       case Letters of
         [L] when L >= $a, L =< $z -> true;
         [] -> false;
-        Letters -> lists:all(fun(L) -> L >= $A andalso L =< $Z end, Letters)
+        Letters -> lists:all(fun(L) -> L >= 0 andalso L =< $Z end, Letters)
```

Comment:
We could make this filter tighter and ensure the sequence is correct, OTOH the tokenizer should have caught an issue already.

Reply:
Anyway can generate any function. For example, I could do `def unquote(:"sigil_\0") do` and we don't want to import that, so we should be strict here too. :)